src/lib/iframe-tools/identity-manager.js

```diff
@@ -87,12 +87,28 @@ export class IdentityManager extends IframeManager {
       return;
     }
 
-    debugLogger.log('[identity-manager:fetchIdentity] Fetch country request.');
-    this.fetchCountry().then((country) => {
+    let shouldReallyCheckCountry;
+    const localStorageKey = 'pp-sdk-shouldReallyCheckCountry';
+    const LS = window.localStorage;
+    if (LS.getItem(localStorageKey) === null) {
+      const randomChance = Math.floor(Math.random() * 100);
+      shouldReallyCheckCountry = randomChance < 50;
+      LS.setItem(localStorageKey, shouldReallyCheckCountry);
+    } else {
+      shouldReallyCheckCountry = LS.getItem(localStorageKey);
+    }
```

Comment:
I like this suggestion and will adopt it ... but the readability suffers a bit .....  because using `shouldReallyCheckCountry` preserves the context throughout the code block ....

Reply:
No problem.  The Honey JS coding standards that our previous team followed avoids `let` when possible in favor of immutability but I'm good with what you originally suggested.